lib/card.rb

```diff
@@ -0,0 +1,18 @@
+class Card
+
+  attr_reader :id
+
+  def initialize(id, attributes)
+    @id = id
+    @attributes = attributes
+  end
+
+  def method_missing(method_name, *args)
```

Comment:
I think it's fine for `Card` to have specific attributes. `Dealer` could have the `CARD_ATTRIBUTES` constant, so adding or removing attributes is an easy change localized to those two classes. I think what I'm really looking for is a nice wrapper around an attributes hash that also has an id.  I already wrote `Permutator` to be independent of the number of attributes so `Card` can really be represented any number of ways.